src/HttpKernel/ContaoCache.php

```diff
@@ -10,11 +10,44 @@
 
 namespace Contao\ManagerBundle\HttpKernel;
 
+use FOS\HttpCache\SymfonyCache\CacheInvalidation;
+use FOS\HttpCache\SymfonyCache\EventDispatchingHttpCache;
```

Comment:
It must, because when `header-replay-bundle` bumps it's dep version, you will inherit the new version here which is potentially conflicting as the class may get altered/removed.

You you must add all packages that provide classes which you use directly in the requires section.